src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Circle circle = new Circle();
+    private final Square square = new Square();
+    private final IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid();
+    private final RightTriangle rightTriangle = new RightTriangle();
+    private final Rectangle rectangle = new Rectangle();
```

Comment:
we do not need these fields

Reply:
i tried but the problem is that later if I just use "new Circle" and etc and use "array[I].setRadius - i cannot use setter. i think I might need help with this

Reply:
Changed it. but still want to know how to fix the problem with "setter" if my next solution is not right.